coordinator/coordinator.go

```diff
@@ -212,17 +311,60 @@ func (c *Coordinator) runSQLTransformationJob(transformSource *metadata.SourceVa
 	c.Logger.Debugw("Created transformation query", "query", query)
 	providerResourceID := provider.ResourceID{Name: resID.Name, Variant: resID.Variant, Type: provider.Transformation}
 	transformationConfig := provider.TransformationConfig{TargetTableID: providerResourceID, Query: query}
-	if err := offlineStore.CreateTransformation(transformationConfig); err != nil {
+	createTransformationConfig := runner.CreateTransformationConfig{
+		OfflineType:          provider.Type(sourceProvider.Type()),
+		OfflineConfig:        sourceProvider.SerializedConfig(),
+		TransformationConfig: transformationConfig,
+		IsUpdate:             false,
+	}
+	serialized, err := createTransformationConfig.Serialize()
+	if err != nil {
+		return err
```

Comment:
Wrap your errors with something like fmt.Errorf("serialize: %w", err).

Makes it 10x faster to find the function that errored